projects/demo/src/modules/components/input-number/input-number.template.html

```diff
@@ -11,37 +11,50 @@
             A component to input numbers. Control value is also of number type.
         </div>
 
-        <p i18n>
-            Component can be used to input money if you choose a currency symbol
-            as a postfix or prefix (see
-            <code>Currency</code>
-            pipe samples).
-        </p>
-
-        <p i18n>
-            There are also other components to input numbers:
-            <a
-                tuiLink
-                routerLink="/components/input-count"
-            >
-                <code>InputCount</code>
-            </a>
-            for integers,
-            <a
-                tuiLink
-                routerLink="/components/slider"
-            >
-                <code>Slider</code>
-            </a>
-            and
-            <a
-                tuiLink
-                routerLink="/components/input-slider"
-            >
-                <code>InputSlider</code>
-            </a>
-            for inputting with a slider
-        </p>
+        <section i18n>
+            <h3>There are also other components to input numbers:</h3>
```

Comment:
btw I remember there was "SeeAlso" component in 1.x taiga that showed related components. Do you know why we don't use it now? 🤔

Reply:
I dont even know about this feature :)

I have investigated this question.
> Do you know why we don't use it now? 

We use it. But it does not work because of the bug (i will fix it in a separate PR).

Reply:
Oh got it, thanks) So I think it's not so important, we live without that feature long time))